### DISCUSSION TITLE
Website Upload

### DISCUSSION THREAD

**Discussion by Spencer021** [gameserverapp/Platform#555] [Suggestions]
Can you add the ability to add custom files to the website? Like for ASA they use Ban list URLs and other various URLS. Can you make it so we can upload those .txt files to the site and and use them in our configs?

**Comment by mvaessen** [comment]
Hi,

Thanks for your suggestions

The banlist url should already be available through config template variables and I'm hoping to add the new admin url soon.

For uploading custom files, please use the self-hosted community website option.
